c7n/resources/account.py

```diff
@@ -1453,3 +1453,45 @@ def get_target(self, resource):
                 return []
             resource[self.annotation][kmskey] = self.data[kmskey]
         return resource[self.annotation]
+
+
+@actions.register('update-glue-datacatalog-encryption')
+class GlueDataCatalogEncryption(BaseAction):
+    """Modifies glue data catalog encryption based on specified
+    parameter.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: data-catalog-encryption
+                resource: account
+                filters:
+                  - type: glue-security-config
+                    CatalogEncryptionMode: DISABLED
+                actions:
+                  - type: update-glue-datacatalog-encryption
+                    attributes:
+                        DataCatalogEncryptionSettings:
```

Comment:
remove the redundant settings wrapper .. using pass through validation doesn't mean not thinking about usability.